golang.mk

```diff
@@ -18,4 +18,5 @@ $(GO_DIR)/bin/go:
 	rm -f $@
 	@echo Downloading and unpacking Go $(GO_VERSION) to $(GO_DIR)
 	curl -s $(GO_DOWNLOAD_URL) | tar xf - --strip-components=1 -C $(GO_DIR)
```

Comment:
The downloaded file is Gzipped:
`GO_DOWNLOAD_URL:=https://dl.google.com/go/go$(GO_VERSION).tar.gz`
which means that the extract should contain `z`:
`curl -s $(GO_DOWNLOAD_URL) | tar zxf - --strip-components=1 -C $(GO_DIR)`